httpie/core.py

```
@@ -0,0 +1,202 @@
+    ])
+
+
+def decode_args(args, stdin_encoding):
+    """
+    Convert all bytes ags to str
+    by decoding them using stdin encoding.
+
+    """
+    return [
+        arg.decode(stdin_encoding)
+        if type(arg) == bytes else arg
```

Ruff (0.12.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)